src/util.ts

```diff
@@ -12,6 +14,19 @@ export const configPathFn = (): string => {
       if (process.env.XDG_CONFIG_HOME) {
         return join(process.env.XDG_CONFIG_HOME, REPLUGGED_FOLDER_NAME);
       }
+      try {
+        if (realUser) {
+          // Get the home directory of the sudo user from /etc/passwd
+          const homeDir = execSync(`getent passwd ${realUser}`, {
+            stdio: [null, null, "ignore"],
+          })
+            .toString("utf-8")
+            .split(":")[5];
```

Comment:
Should still verify this has a banks value and it is a valid path. If parsing fails, we want it to stop.

Reply:
i highly doubt you could even run a unix system if this is misconfigured, but i handled it anyway